apps/liteprotocoltester/Dockerfile.liteprotocoltester.compile

```diff
@@ -43,15 +42,30 @@ EXPOSE 30303 60000 8545
 # Referenced in the binary
 RUN apk add --no-cache libgcc pcre-dev libpq-dev \
   wget \
-  iproute2
+  iproute2 \
+  python3
 
 # Fix for 'Error loading shared library libpcre.so.3: No such file or directory'
 RUN ln -s /usr/lib/libpcre.so /usr/lib/libpcre.so.3
 
 COPY --from=nim-build /app/build/liteprotocoltester /usr/bin/
+RUN chmod +x /usr/bin/liteprotocoltester
+
+# Standalone image to be used manually and in lpt-runner -------------------------------------------
+FROM base_lpt AS standalone_lpt
+
 COPY --from=nim-build /app/apps/liteprotocoltester/run_tester_node.sh /usr/bin/
+RUN chmod +x /usr/bin/run_tester_node.sh
 
 ENTRYPOINT ["/usr/bin/run_tester_node.sh", "/usr/bin/liteprotocoltester"]
 
-# # By default just show help if called without arguments
-CMD ["--help"]
+# Image for infra deployment -------------------------------------------
+FROM base_lpt AS deployment_lpt
+
+COPY --from=nim-build /app/apps/liteprotocoltester/run_tester_node_at_infra.sh /usr/bin/
+COPY --from=nim-build /app/apps/liteprotocoltester/infra.env /usr/bin/
```

Comment:
You can just define the env variables in this Dockerfile and set their default values. They can be changed if you pass them to the container when starting it.